Content.Server/Explosion/Components/ShockOnTriggerComponent.cs

```diff
@@ -0,0 +1,39 @@
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Server.Explosion.Components;
+
+/// <summary>
+/// A component that electrocutes an entity having this component when a trigger is triggered.
+/// </summary>
+[RegisterComponent, AutoGenerateComponentPause]
+public sealed partial class ShockOnTriggerComponent : Component
+{
+    /// <summary>
+    /// The force of an electric shock when the trigger is triggered.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField]
+    public int Force = 5;
```

Comment:
Also yes, change it to DamageSpecifier

Reply:
> Also yes, change it to DamageSpecifier

Just now I noticed that I understood you literally and just renamed the data field. The TryDoElectrocution method does not accept a DamageSpecifier, it only accepts an integer value. And I'd rather not change this in TryDoElectrocution, it's too difficult for me to change all the uses of the method so that everything supports DamageSpecifier. I will return the name back to Damage.